src/plugins/dataflow/components/dataflow-vars.sass

```diff
@@ -92,3 +92,13 @@ $data-green: #49d381
 $dataflow-topbar-height: 40px
 $dataflow-toolbar-width: 84px
 $dataflow-app-header: 56px
+
+//
+// Link button
+//
+$charcoal-light-1: #949494
+$charcoal-light-4: #dfdfdf
+$workspace-teal-light-2: #6fc6da
+$workspace-teal-light-4: #b7e2ec
```

Comment:
These variables are all defined in `src/components/vars.sass`. You should not duplicate them here. I think you should be able to access them by importing `vars.sass` at the top of this file: `@import ../../../components/vars`.

If that doesn't work here, you should import `vars.sass` in `dataflow-program-link-table-button.scss`, where the variables are actually be used.